README.md

```diff
@@ -1,2 +1,94 @@
 # massa-wallet-provider
-Massa's wallet provider library
+
+`Massa-wallet-provider` is a TypeScript library that one could utilize to establish a connection between a frontend dapp application and any browser wallet extensions that implement the [massa standard](https://github.com/massalabs/massa-standards/blob/main/wallet/dapps-communication.md). With this library one can access all massa-blockchain wallets and their standardized functionalities for ready use.
+
+## Usage
+
+`Massa-wallet-provider` could be used as a library for frameworks or as a stand-alone bundled js file which can be easily loaded into the browser.
+
+### Library (Node.js/React/Vue.js) usage
+
+> npm install @massalabs/massa-wallet-provider
+
+### Browser usage
+
+Add the following script to your html file:
+
+```ts
+<script
+    type="text/javascript"
+    src="https://cdn.jsdelivr.net/npm/@massalabs/massa-wallet-provider@x.x.x/bundle.js"
+></script>
+```
+
+whereby the x.x.x is one of the available released versions under
+[Massa-web3's releases page](https://github.com/massalabs/massa-wallet-provider/releases):
```

Comment:
wrong copy/paste ?

Reply:
yep, copied some boilerplate from the web3 stuff. Fixed it now.